src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureArea,DrawFIgure {
```

Comment:
Interface names should not be verbs. In this case, `DrawFigure` should be renamed to something that represents what the implementing class is, such as `Drawable` or similar.